safe/gui/widgets/dock.py

```diff
@@ -915,7 +914,7 @@ def layer_changed(self, layer):
             if show_keywords:
                 if inasafe_keyword_version_key not in keywords.keys():
                     show_keyword_version_message(
-                        self, 'No Version', self.inasafe_version)
```

Comment:
Is this block still valid? I think this was legacy and I was wondering if this block can be reached again today